src/components/DeploymentForm.vue

```diff
@@ -119,14 +134,21 @@
   const { value: workPoolName } = useField<string | null>('workPoolName')
   const { value: workQueueName } = useField<string | null>('workQueueName')
   const { value: tags } = useField<string[] | null>('tags')
+  const { value: infrastructureOverrides } = useField<string>('infrastructureOverrides')
 
   const emit = defineEmits<{
     (event: 'submit', value: DeploymentUpdate): void,
     (event: 'cancel'): void,
   }>()
 
   const submit = handleSubmit((values) => {
-    emit('submit', values)
+    try {
+      JSON.parse(infrastructureOverrides.value)
```

Comment:
Is this parse attempting to validate the json? We have a validation rule for that which we should use for this sort of thing.

Reply:
https://github.com/PrefectHQ/prefect-ui-library/blob/main/src/utilities/validation.ts#L246